finished/src/app/headers.ts

```diff
@@ -21,12 +21,12 @@ export const setCommonHeaders =
     // Explicitly disables access to specific browser features/APIs
     headers.set(
       "Permissions-Policy",
-      "geolocation=(), microphone=(), camera=()",
+      "geolocation=(), microphone=(), camera=()"
     );
 
     // Defines trusted sources for content loading and script execution:
     headers.set(
       "Content-Security-Policy",
-      `default-src 'self'; script-src 'self' 'nonce-${nonce}' https://challenges.cloudflare.com; style-src 'self' 'unsafe-inline'; frame-src https://challenges.cloudflare.com; object-src 'none';`,
+      `default-src 'self'; script-src 'self' 'nonce-${nonce}' https://challenges.cloudflare.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://applywize.redwoodjs.workers.dev; font-src 'self' https://fonts.gstatic.com; frame-src https://challenges.cloudflare.com; object-src 'none';`
```

Comment:
We need to allow google fonts links in CSP policy. We could consider making default CSP policy we add in starters more permissive for styles and fonts, I think scripts are the bigger deal - would probably make for less headaches in tutorial.